database/factories/PrintJobFactory.php

```diff
@@ -13,8 +14,7 @@ public function definition()
     {
         return [
```

Comment:
PrintJob requires a `printer_id` attribute to be added during seeding as there is no default value.